src/core/bpf-cgroup-program-spec-parsers.c

```diff
@@ -62,6 +63,69 @@ static int create_bpf_cgroup_program_spec(
         return 0;
 }
 
+static int get_json_string_value(JsonVariant *json, const char* key, const char **value) {
+        JsonVariant *var;
+
+        var = json_variant_by_key(json, key);
+        if (!var)
+                return -ENOENT;
+
+        *value = json_variant_string(var);
+        if (!*value)
+                return -EINVAL;
```

Comment:
we generally don't clobber return values on error, see https://systemd.io/CODING_STYLE/

(that said, I am not convinced we want to use JSON for this)

Reply:
Switched to `<attach_type_str>:/path/in/bpffs` like suggested.